umm-lib/test/cmr/umm/test/echo10/granule.clj

```diff
@@ -38,7 +43,9 @@
     (let [xml (echo10/umm->echo10-xml granule)
           parsed (g/parse-granule xml)
           expected-parsed (umm->expected-parsed-echo10 granule)]
-      (= parsed expected-parsed))))
+      ;; Remove crid-ids and feature-ids because they are not supported in echo10 
+      (= (remove-crid-ids-and-feature-ids parsed)
```

Comment:
We should be making all modifications in `umm->expected-parsed-echo10`.

Reply:
Moved it to umm->expected-parsed-echo10 for expected-parsed. But for parsed, we still need to call the function because it's possible that parsed contains :crid-ids nil :feature-ids nil.